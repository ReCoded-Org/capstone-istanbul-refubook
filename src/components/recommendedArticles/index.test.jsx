import React from 'react';
import RecommendedArticles from './';
import renderer from 'react-test-renderer';
it('RecommendedArticles renders correctly', () => {
  const tree = renderer.create(<RecommendedArticles />).toJSON();
  expect(tree).toMatchSnapshot();
});
