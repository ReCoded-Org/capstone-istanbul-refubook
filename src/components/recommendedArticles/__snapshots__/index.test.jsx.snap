// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecommendedArticles renders correctly 1`] = `
<div
  className="grid grid-cols-2 lg:grid-cols-3 xl:grid-cols-3 gap-4 "
>
  <div
    className="py-8 px-4 flex justify-center items-center"
  >
    <div
      className="max-w-sm rounded overflow-hidden shadow-md"
    >
      <img
        alt="Sunset in the mountains"
        className="w-full"
        src="https://via.placeholder.com/350x200"
      />
      <div
        className="px-6 py-4"
      >
        <div
          className="font-bold text-xl mb-2"
        >
          <a
            href="/"
          >
            How I learned Turkish quickly and what are the best places to learn.
          </a>
        </div>
        <p
          className="text-gray-700 text-base"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil.
        </p>
      </div>
      <div
        className="px-6 py-4 flex items-center"
      >
        <img
          alt="Avatar of Ahmad Faysal"
          className="w-10 h-10 rounded-full mr-4"
          src="https://via.placeholder.com/150"
        />
        <div
          className="text-sm"
        >
          <p
            className="text-gray-900 leading-none"
          >
            Ahmad Faysal
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-8 px-4 flex justify-center items-center"
  >
    <div
      className="max-w-sm rounded overflow-hidden shadow-md"
    >
      <img
        alt="Sunset in the mountains"
        className="w-full"
        src="https://via.placeholder.com/350x200"
      />
      <div
        className="px-6 py-4"
      >
        <div
          className="font-bold text-xl mb-2"
        >
          <a
            href="/"
          >
            How I learned Turkish quickly and what are the best places to learn.
          </a>
        </div>
        <p
          className="text-gray-700 text-base"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil.
        </p>
      </div>
      <div
        className="px-6 py-4 flex items-center"
      >
        <img
          alt="Avatar of Ahmad Faysal"
          className="w-10 h-10 rounded-full mr-4"
          src="https://via.placeholder.com/150"
        />
        <div
          className="text-sm"
        >
          <p
            className="text-gray-900 leading-none"
          >
            Ahmad Faysal
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-8 px-4 flex justify-center items-center"
  >
    <div
      className="max-w-sm rounded overflow-hidden shadow-md"
    >
      <img
        alt="Sunset in the mountains"
        className="w-full"
        src="https://via.placeholder.com/350x200"
      />
      <div
        className="px-6 py-4"
      >
        <div
          className="font-bold text-xl mb-2"
        >
          <a
            href="/"
          >
            How I learned Turkish quickly and what are the best places to learn.
          </a>
        </div>
        <p
          className="text-gray-700 text-base"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil.
        </p>
      </div>
      <div
        className="px-6 py-4 flex items-center"
      >
        <img
          alt="Avatar of Ahmad Faysal"
          className="w-10 h-10 rounded-full mr-4"
          src="https://via.placeholder.com/150"
        />
        <div
          className="text-sm"
        >
          <p
            className="text-gray-900 leading-none"
          >
            Ahmad Faysal
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-8 px-4 flex justify-center items-center"
  >
    <div
      className="max-w-sm rounded overflow-hidden shadow-md"
    >
      <img
        alt="Sunset in the mountains"
        className="w-full"
        src="https://via.placeholder.com/350x200"
      />
      <div
        className="px-6 py-4"
      >
        <div
          className="font-bold text-xl mb-2"
        >
          <a
            href="/"
          >
            How I learned Turkish quickly and what are the best places to learn.
          </a>
        </div>
        <p
          className="text-gray-700 text-base"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil.
        </p>
      </div>
      <div
        className="px-6 py-4 flex items-center"
      >
        <img
          alt="Avatar of Ahmad Faysal"
          className="w-10 h-10 rounded-full mr-4"
          src="https://via.placeholder.com/150"
        />
        <div
          className="text-sm"
        >
          <p
            className="text-gray-900 leading-none"
          >
            Ahmad Faysal
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-8 px-4 flex justify-center items-center"
  >
    <div
      className="max-w-sm rounded overflow-hidden shadow-md"
    >
      <img
        alt="Sunset in the mountains"
        className="w-full"
        src="https://via.placeholder.com/350x200"
      />
      <div
        className="px-6 py-4"
      >
        <div
          className="font-bold text-xl mb-2"
        >
          <a
            href="/"
          >
            How I learned Turkish quickly and what are the best places to learn.
          </a>
        </div>
        <p
          className="text-gray-700 text-base"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil.
        </p>
      </div>
      <div
        className="px-6 py-4 flex items-center"
      >
        <img
          alt="Avatar of Ahmad Faysal"
          className="w-10 h-10 rounded-full mr-4"
          src="https://via.placeholder.com/150"
        />
        <div
          className="text-sm"
        >
          <p
            className="text-gray-900 leading-none"
          >
            Ahmad Faysal
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-8 px-4 flex justify-center items-center"
  >
    <div
      className="max-w-sm rounded overflow-hidden shadow-md"
    >
      <img
        alt="Sunset in the mountains"
        className="w-full"
        src="https://via.placeholder.com/350x200"
      />
      <div
        className="px-6 py-4"
      >
        <div
          className="font-bold text-xl mb-2"
        >
          <a
            href="/"
          >
            How I learned Turkish quickly and what are the best places to learn.
          </a>
        </div>
        <p
          className="text-gray-700 text-base"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil.
        </p>
      </div>
      <div
        className="px-6 py-4 flex items-center"
      >
        <img
          alt="Avatar of Ahmad Faysal"
          className="w-10 h-10 rounded-full mr-4"
          src="https://via.placeholder.com/150"
        />
        <div
          className="text-sm"
        >
          <p
            className="text-gray-900 leading-none"
          >
            Ahmad Faysal
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
